website/docs/network-traffic.mdx

```diff
@@ -91,28 +91,28 @@ Runs on the customer's own servers. We suggest <a href="https://configcat.com/su
 
 ### Avoid keeping lots of data in the comparison value of Targeting Rules or segments
 
-The comparison value of a Targeting Rule or segment is stored in the `config JSON` and downloaded by the SDKs. If you have a lot of Targeting Rules or segments with long comparison values, you can lower the size of the `config JSON` by shortening them.
+The comparison value of a Targeting Rule or segment is stored in the `config JSON` and downloaded by the SDKs. If you have a lot of Targeting Rules or segments with lengthy comparison values, you can reduce the size of the `config JSON` by shortening them.
 
 ### Consider the amount of text you keep in a text setting's value
 
-Similarly to the comparison value of Targeting Rules or segments, the value of a text setting is stored in the `config JSON` and downloaded by the SDKs. If you have a lot of text settings with long values, you can lower the size of the `config JSON` by shortening them.
+Similarly to the comparison value of Targeting Rules or segments, the value of a text setting is stored in the `config JSON` and downloaded by the SDKs. If you have a lot of text settings with lengthy values, you can reduce the size of the `config JSON` by shortening them.
 
 ### Separate your feature flags into multiple configs
 
-If you have a lot of feature flags, you can reduce the size of the `config JSON` by separating them into multiple configs. This way, the payload of each download will be smaller.
+If you have a large number of feature flags, you can reduce the size of the `config JSON` by separating them into multiple configs. This way, the payload of each download will be smaller.
 
 ### Use the ConfigCat Proxy to cache/proxy config JSON downloads
 
-The [ConfigCat Proxy](https://configcat.com/docs/advanced/proxy/proxy-overview/) allows you to host a feature flag evaluation and config JSON proxy/cache service within your own infrastructure.  
-The [CDN proxy](https://configcat.com/docs/advanced/proxy/endpoints/#cdn-proxy) feature allows you to proxy/cache config JSON downloads and the ConfigCat SDKs can be routed to the ConfigCat Proxy running in your own infrastructure greatly reducing network traffic.
+The [ConfigCat Proxy](/advanced/proxy/proxy-overview/) allows you to host a feature flag evaluation and config JSON proxy/cache service within your own infrastructure.  
+The [CDN proxy](/advanced/proxy/endpoints/#cdn-proxy) feature allows you to proxy/cache config JSON downloads, and the ConfigCat SDKs can be routed to the ConfigCat Proxy running in your own infrastructure greatly reducing network traffic.
 
 ### Purchase additional quota
 
-You can purchase additional quota for your subscription to increase your limits. [Contact us](https://configcat.com/support/) for more information.
+You can purchase an additional quota for your subscription to increase your limits. [Contact us](https://configcat.com/support/) for more information.
```

Comment:
Hmm, I'm not sure the word "quota" means what we want to mean in this context. Wouldn't something like "quota add-on" or simply "add-on" be better? @aisz0811 What's your opinion?